IG.md

```diff
@@ -0,0 +1,45 @@
+# Scripting in [ForgeRock Identity Gateway](https://www.forgerock.com/platform/identity-gateway) (IG)
+
+## IG in ForgeOps
+
+IG is not included in the `cdk` profile we've been using so far. It is used, however, for OAuth 2.0 API protection in the `oauth2` profile in the 6.5 version of the ForgeOps configuration. To have the `oauth2` profile deployed, stop your current deployment with `Ctrl^C` if it is still running in the foreground, or run `skaffold delete` if the deployment is not active in your terminal. Then, clear the persistent volumes with `kubectl delete pvc --all`. Optionally, you can delete your VM with `minikube delete` and [create a new one](https://backstage.forgerock.com/docs/forgeops/6.5/devops-guide-minikube/#devops-implementation-env-cluster).
```

Comment:
Using `bin/clean.sh` is probably better than `kubectl delete pvc --all`

Reply:
OK, thanks. By the way, this is an unfinished section. I do welcome any comments, just want you to be aware the content will likely change.

Reply:
I will probably use 6.5 throughout.